packages/components/src/styles/media/media.stories.ts

```diff
@@ -1,21 +1,25 @@
 import '../../solid-components';
-
+import { html } from 'lit-html';
 import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
 
 const { argTypes, parameters } = storybookDefaults('sd-media');
 const { overrideArgs } = storybookHelpers('sd-media');
 const { generateTemplate } = storybookTemplate('sd-media');
 
 /**
- * Add a media caption to the bottom of an image.
+ * Used to display an image or a video preview.
+ *
+ *  * **Related templates**:
+ * - [Media](?path=/docs/templates-media--docs)
  */
 export default {
   title: 'Styles/sd-media',
+  tags: ['!dev'],
   component: 'sd-media',
   parameters: {
     ...parameters
   },
-  args: overrideArgs({ type: 'slot', name: 'default', value: 'Lorem Ipsum' }),
+  args: overrideArgs({ type: 'slot', name: 'default', value: 'Default' }),
```

Comment:
Leave default slot empty, pls

Reply:
Don’t show default story ... we already show that in the intro.

Reply:
A fixed for it was added to docs branch, this way all styles components will have the default story hidden

Reply:
I can still see it
![Bildschirmfoto 2024-09-06 um 09 15 14](https://github.com/user-attachments/assets/7dd803c5-310e-47e4-96e5-6f1fcd1ae73f)

Reply:
Will be fixed in docs but in this branch is not possible to see already. [PR for the fix in docs](https://github.com/solid-design-system/solid/pull/1359)